torch/_subclasses/fake_tensor.py

```diff
@@ -658,24 +705,29 @@ def to_real_tensor(e):
 
         for e in tree_flatten((args, kwargs))[0]:
             if isinstance(e, torch.Tensor):
-                storages.add(e.storage()._cdata)
+                if not e.is_sparse:
+                    storages.add(e.storage()._cdata)
```

Comment:
I would maybe add a `# TODO: sparse storage aliasing` here

Reply:
ok will do in follow up